portalnet/src/discovery.rs

```diff
@@ -36,7 +37,7 @@ const TALKREQ_CHANNEL_BUFFER: usize = 100;
 /// ENR key for portal network client version.
 pub const ENR_PORTAL_CLIENT_KEY: &str = "c";
 
-pub type ProtocolRequest = Vec<u8>;
+pub type ProtocolRequest = Bytes;
```

Comment:
After looking more into how this is used, I think it's better if this is not changed to `Bytes` and left as `Vec<u8>`.